.gitignore

```diff
@@ -4,8 +4,12 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
-# Secrets
-/config/secrets.yml
+# Ignore all .env files
+/.env*
+
+# Ignore all /public/packs and /public/packs-test files
+/public/packs/*
+/public/packs-test/*
```

Comment:
Try:
```
public/packs
public/packs-test
```

These two guys mustn't go into repo :) Every developer should have their own copy of the packs

P.S.
+40K seems to much, doesn't it :) ?
<img width="154" alt="screen shot 2018-02-22 at 10 24 16 pm" src="https://user-images.githubusercontent.com/9846273/36559462-28f9865e-181f-11e8-9e39-0718088310e6.png">